Editor/GraphView.cs

```diff
@@ -64,6 +65,24 @@ private static void Draw(Rect area, ref List<KeyValuePair<DateTime, float>> data
 
             // grid
             Handles.color = new Color(1f, 1f, 1f, 0.5f);
+
+
+            var unitCount = 1;
+            var unitIndex = 0;
+            var current = maxValue;
+
+            while (current >= 1000)
+            {
+                current /= 1000;
+                unitCount *= 1000;
+                unitIndex++;
+            }
+
+            if (unitIndex > unitStr.Length - 1)
+            {
+                unitIndex = unitStr.Length - 1;
+            }
+
```

Comment:
It might be better this routine is static method.

Reply:
This method (Draw) is static method.

Does that mean I should cut out function?

Reply:
Yes, it make the operation clarify.